easybuild/easyconfigs/a/annovar/annovar-20200607-GCCcore-12.3.0.eb

```
@@ -0,0 +1,38 @@
+# J. Sassmannshausen /NHS/GSTT: toolchain updated to GCCcore-11.2
+
+easyblock = 'Tarball'
+
+name = 'annovar'
+version = '20200607'  # version reported by `annotate_variation.pl -h`
+
+homepage = 'http://annovar.openbioinformatics.org/en/latest/'
+description = """ANNOVAR is an efficient software tool to utilize update-to-date information 
+to functionally annotate genetic variants detected from diverse genomes (including human 
+genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others)."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+download_instructions = """
+Registration required for download
+via http://www.openbioinformatics.org/annovar/annovar_download_form.php
+rename after download to '%(name)s-%(version)s.tar.gz'
```

Using templates in `download_instructions` results in failure in easyconfigs test suite, so let's just hardcode the version then...
```suggestion
rename after download to 'annovar-20200607.tar.gz'
```